src/app/components/UsefulLinks/README.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+Renders a set of Useful Links within a section element. Each list item contains a link with a title. These elements sit within a region landmark.
+The link has a tap area. The component uses the curation `visualStyle` of `LINKS` and `visualProminence` of `LOW`.
+
+## Props
+
+| Name      | type      | Description                                          |
+| --------- | --------- | ---------------------------------------------------- |
+| title     | string    | The H2 title of the useful links section             |
+| position  | number    | The position of the curation item                    |
```

Comment:
Can we either remove `position`, or update it with `id`, update the description & state that it is optional?